chromium_src/components/permissions/permission_util.cc

```diff
@@ -55,6 +59,10 @@ std::string PermissionUtil::GetPermissionString(
 // static
 bool PermissionUtil::GetPermissionType(ContentSettingsType type,
                                        blink::PermissionType* out) {
+  if (type == ContentSettingsType::BRAVE_GOOGLE_SIGN_IN) {
```

Comment:
pls keep the order everywhere. `ContentSettingsType::BRAVE_GOOGLE_SIGN_IN` goes after `BRAVE_ETHEREUM/BRAVE_SOLANA`.